src/schema/datatypes/anat.yaml

```diff
@@ -28,6 +23,39 @@
     rec: optional
     part: optional
 # Second group
+- suffixes:
+    - T1map
+    - T2map
+    - T2starmap
+    - R1map
+    - R2map
+    - R2starmap
+    - PDmap
+    - MTRmap
+    - MTsat
+    - UNIT1
+    - T1rho
+    - MWFmap
+    - MTVmap
+    - PDT2map
+    - Chimap
+    - TB1map
+    - RB1map
+    - S0map
+    - M0map
+  extensions:
+    - .nii.gz
+    - .nii
+    - .json
+  entities:
+    sub: required
+    ses: optional
+    run: optional
+    acq: optional
+    ce: optional
+    rec: optional
+    part: optional
```

Comment:
Apologies for the naive question, but aren't at least some of these (e.g., `Chimap`) _derived_ from complex data? Is `part` applicable in those cases?

Reply:
Yes, derivation of `chimap` often includes phase data, but not the map itself. So `part` is required for `QSM` file collections, but not for its outputs.

Reply:
`part` is labeled as optional here, though. Is it technically possible to have a complex-valued `Chimap`? I.e., should `part` even be optional for this suffix?

Reply:
@tsalo I was about to ask you what happens if I omitted `part` from the list for the maps group, as it is not relevant. I can see cases where `rec` or `ce` can be useful for maps, but not for part.

Reply:
You should define as many groups as there are unique combinations of required and optional entities, so if `part` is invalid for `Chimap`, but optional for `S0map`, then you need at least two groups of suffixes.

The immediate effect of splitting it will be that there will be an extra line in the entity table. One will have a list of suffixes where `part` is optional and one will have those where it's prohibited. Ultimately, though, this will be used to determine if BIDS datasets are actually valid, so if there is a `Chimap` with `part` the validator would flag it as an impossible combination.

Reply:
@tsalo susceptibility maps are given in `p.p.m`, so I don't think that there are cases where they can be complex. The same goes for the majority of the maps I can think of. Unlikely, but if at some point it happens to become the case for some parametric maps, then they can be moved to a separate group. Until then, I will drop `part` from maps.

Reply:
9c3e0e8